ui/App/DesignSystemGuideModal.svelte

```diff
@@ -732,7 +732,7 @@
         </div>
 
         <div class="swatch">
-          <Tooltip value="Bottom" position="bottom">
+          <Tooltip value="A very long text" position="bottom">
```

Comment:
Can we make all the example swatches have the same text?

Reply:
I wanted to have the text for the top position shorter than the container so I only updated this one.